instructionAPI/h/Instruction.h

```diff
@@ -48,281 +48,288 @@
 
 namespace Dyninst
 {
```

Comment:
Undo the the reindentation of this file.  Reindenting a file (that wasn't confusingly indented) just added noise to the commit and obfuscates the real changes.  Diffing

``` 
  instructionAPI/h/Instruction.h    | 541 ++++++++++++++++++++--------------------
  instructionAPI/h/Operation_impl.h | 236 +++++++++---------
```

If I ignore whitespace changes the diff shows only 2% of the lines changed are real:

```
  instructionAPI/h/Instruction.h    | 7 +++++++
  instructionAPI/h/Operation_impl.h | 8 ++++++++
```